libs/vectorise/include/vectorise/memory/array.hpp

```diff
@@ -54,7 +54,7 @@ class Array : public VectorSlice<T, type_size>
 
     if (n > 0)
     {
-      this->pointer_ = (type *)_mm_malloc(this->padded_size() * sizeof(type), 32);
+      this->pointer_ = static_cast<type *>(_mm_malloc(this->padded_size() * sizeof(type), 32));
```

Comment:
Think this should just be reinterpret cast

Reply:
done